packages/ag-charts-community/src/chart/series/hierarchy/hierarchySeries.test.ts

```diff
@@ -0,0 +1,113 @@
+import type { BBox } from '../../../scene/bbox';
+import type { ChartAxisDirection } from '../../chartAxisDirection';
+import type { ChartLegendDatum, ChartLegendType } from '../../legendDatum';
+import type { SeriesNodeDataContext } from '../series';
+import type { SeriesTooltip } from '../seriesTooltip';
+import { HierarchySeries } from './hierarchySeries';
+
+class ExampleHierarchySeries extends HierarchySeries<any> {
+    override tooltip: SeriesTooltip<any> = null!;
+
+    override getSeriesDomain(_direction: ChartAxisDirection): any[] {
+        throw new Error('Method not implemented.');
+    }
+
+    override createNodeData(): Promise<SeriesNodeDataContext<any, any>[]> {
+        throw new Error('Method not implemented.');
+    }
+
+    override update(_opts: { seriesRect?: BBox | undefined }): Promise<void> {
+        throw new Error('Method not implemented.');
+    }
+
+    override getTooltipHtml(_seriesDatum: any): string {
+        throw new Error('Method not implemented.');
+    }
+
+    override getLegendData(_legendType: unknown): ChartLegendDatum<any>[] | ChartLegendDatum<ChartLegendType>[] {
+        throw new Error('Method not implemented.');
+    }
+}
+
+describe('hierarchySeries', () => {
```

Comment:
Nit: Maybe `HierarchySeries` for consistency with the class name capitalisation rather than the module name?